docs/customization/model.rst

```diff
@@ -274,99 +295,119 @@ Just like for regular models you can also check the class of translatable models
 
     $ php bin/console debug:container --parameter=sylius.model.shipping_method_translation.class
 
-**1.** In order to add a translatable property to your entity you need to define it on the ``App\Entity\ShippingMethodTranslation`` class of your bundle, that will extend the base ``Sylius\Component\Shipping\Model\ShippingMethodTranslation``.
+**1.** In order to add a translatable property to your entity, start from defining it on the ``src\Entity\Shipping\ShippingMethodTranslationInterface`` interface, that will extend the base ``Sylius\Component\Shipping\Model\ShippingMethodTranslationInterface``.
+
+.. code-block:: php
+
+    <?php
+
+    declare(strict_types=1);
+
+    namespace App\Entity\Shipping;
+
+    use Sylius\Component\Shipping\Model\ShippingMethodTranslationInterface as BaseShippingMethodTranslationInterface;
+
+    interface ShippingMethodTranslationInterface extends BaseShippingMethodTranslationInterface
+    {
+        public function getDeliveryConditions(): ?string;
+
+        public function setDeliveryConditions(?string $deliveryConditions): void;
+    }
+
+**2.** Having the property added on the interface, you can proceed to add in on the entity itself. The class `src\Entity\Shipping\ShippingMethodTranslation` is already there in the right place.
 
 .. code-block:: php
 
     <?php
 
-    namespace App\Entity;
+    declare(strict_types=1);
+
+    namespace App\Entity\Shipping;
 
+    use Doctrine\ORM\Mapping as ORM;
     use Sylius\Component\Shipping\Model\ShippingMethodTranslation as BaseShippingMethodTranslation;
 
-    class ShippingMethodTranslation extends BaseShippingMethodTranslation
+    /**
+     * @ORM\MappedSuperclass()
+     * @ORM\Table(name="sylius_shipping_method_translation")
+     */
+    class ShippingMethodTranslation extends BaseShippingMethodTranslation implements ShippingMethodTranslationInterface
     {
-        /**
-         * @var string
-         */
+        /** @ORM\Column(type="string",nullable=true) */
         private $deliveryConditions;
 
-        /**
-         * @return string
-         */
-        public function getDeliveryConditions(): string
+        public function getDeliveryConditions(): ?string
         {
             return $this->deliveryConditions;
         }
 
-        /**
-         * @param string $deliveryConditions
-         */
-        public function setDeliveryConditions(string $deliveryConditions): void
+        public function setDeliveryConditions(?string $deliveryConditions): void
         {
             $this->deliveryConditions = $deliveryConditions;
         }
     }
 
-**2.** Next define your translation entity's mapping.
+**3.** Add the field's getters and setters on the ``src\Entity\Shipping\ShippingMethodInterface`` interface.
```

Comment:
```suggestion
**3.** Add the field's getters and setters on the ``App\Entity\Shipping\ShippingMethodInterface`` interface.
```

Reply:
the convention we have used was providing the path to file here, and the path is with `src/` I suppose.

Reply:
Is there a convention? https://docs.sylius.com/en/1.4/customization/menu.html uses class names, https://docs.sylius.com/en/1.4/customization/form.html uses filenames, then https://docs.sylius.com/en/1.4/cookbook/entities/custom-translatable-model.html uses class names again etc.

IMO the best solution is using fully-qualified class names in textual documentation and writing the path of the file inside a code snippet as a comment. Symfony uses this approach - https://symfony.com/doc/current/controller.html.